### ISSUE TITLE
Enabling `vendor` in `deno.json` results in esbuild errors

### ISSUE THREAD

**Issue by A1029384756** [denoland/fresh#2796]
I am attempting  to package a Fresh application with Nix which disables all internet access at build time. As a result, I vendor my dependencies by setting `"vendor": true` in `deno.json`.  Once this is done, I get the following error:
```
✘ [ERROR] [unreachable] Not an ESM module. [plugin deno-loader]
```
in multiple spots. Is that something that is known or is more work needed to find the source of this issue?

This issue can be somewhat inconsistent as a build will succeed if a build has already run without running `deno clean`

**Comment by mabasic** [comment]
Same as this issue: https://github.com/denoland/fresh/issues/2606

**Comment by marvinhagemeister** [comment]
Duplicate of https://github.com/denoland/fresh/issues/2606
